lib/routes/account.js

```diff
@@ -1914,8 +1995,162 @@ module.exports = function (
             function () {
               reply({})
             },
+            function (err) {
+              reply(err)
+            }
+          )
+      }
+    },
+    {
+      method: 'GET',
+      path: '/recovery_email/emails',
```

Comment:
API nit: I suggest `/recovery_emails` for this, by analogy to the existing `/account/devices` and `/account/device/*` endpoints (which in retrospect, could really have just been `/devices` and `/device/*`.